src/Microsoft.Android.Sdk.ILLink/BaseMarkHandler.cs

```diff
@@ -0,0 +1,18 @@
+using Mono.Linker.Steps;
+using Microsoft.Android.Sdk.ILLink;
+
+namespace Mono.Linker
+{
+	public class BaseMarkHandler : IMarkHandler
+	{
+		protected LinkContext Context;
+		protected AnnotationStore Annotations => Context?.Annotations;
+		protected LinkContextMetadataResolver resolver;
```

Comment:
Should this field instead be of type `IMetadataResolver`?

Reply:
Makes sense, fixed.